api/interfaces.go

```diff
@@ -12,4 +13,8 @@ import (
 type MongoServer interface {
 	Close(ctx context.Context) error
 	Checker(ctx context.Context, state *healthcheck.CheckState) error
+	GetImages(ctx context.Context, collectionID string) ([]models.Image, error)
+	GetImage(id string) (*models.Image, error)
+	UpdateImage(ctx context.Context, id string, image *models.Image) error
+	UpsertImage(id string, image *models.Image) (err error)
```

Comment:
Some methods have context passed and some don't. Should there be some additional logging added / removed?

Reply:
yes, maybe we should remove the logs from MongoDB, as the caller already logs errors. I will remove the logs from mongo.go

Reply:
I can provide ctx and info logs for other methods.